### PR TITLE
feat(web): create a layout wrapper with a navbar

### PR DESCRIPTION
**Pull request by YiseBoge**
image

### CODE REVIEW

File: starter-project-web-react/components/Layout.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import NavBar from './NavBar'
+
+const Layout = ({children}) => {
+  return (
+    <div>
```
Comment (abdi-edoc-de) [inline starter-project-web-react/components/Layout.jsx:6]: this div is doing nothing.

File: starter-project-web-react/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+
+export default function ButtonAppBar() {
+  const pages = ['Products', 'Pricing', 'Blog'];
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static">
+        <Toolbar>
+          <IconButton
```
Comment (abdi-edoc-de) [inline starter-project-web-react/components/NavBar.jsx:16]: is this button needed if we don't have a menu?

**Review by abdi-edoc-de — APPROVED** [review]
I have added some minor things to fix from what you told me earlier.

**Comment by YiseBoge** [comment]
> I have added some minor things to fix from what you told me earlier.

Nice

**Pull request merged**